tests/console/yast2_i.pm

```diff
@@ -81,20 +81,13 @@ sub run() {
         send_key 'alt-o';
     }
 
-    # Whether summary is shown depends on PKGMGR_ACTION_AT_EXIT in /etc/sysconfig/yast2
```

Comment:
but I'm not sure we're actually testing upgrades of such a system. sle11 and 11.X will have this set. Not sure when the default was changed.